### PR TITLE
Add atom editors for new explainer types

### PR DESCRIPTION
**Pull request by regiskuckaertz**
- also bumped atom-maker version and fixed a broken reference
- added a new `textarea` input type for longer texts
- minor fix in how cards are laid out on the index page

### CODE REVIEW

File: app/util/AtomElementBuilders.scala [hunk 1]
```diff
@@ -4,6 +4,10 @@ import com.gu.contentatom.thrift.atom.cta.CTAAtom
 import com.gu.contentatom.thrift.atom.explainer.{DisplayType, ExplainerAtom}
 import com.gu.contentatom.thrift.atom.recipe.{RecipeAtom, Tags => RecipeTags, Time => RecipeTime}
 import com.gu.contentatom.thrift.atom.storyquestions.{RelatedStoryLinkType, StoryQuestionsAtom}
+import com.gu.contentatom.thrift.atom.qanda.{QAndAAtom, QAndAItem}
```
Comment (shaundillon) [inline app/util/AtomElementBuilders.scala]: It looks like the build is failing because it can't find all of these.

**Comment by marialivia16** [comment]
Looks good but make sure you bump up the `atom-maker` libs as well to make it compile 😃

**Comment by regiskuckaertz** [comment]
Thanks guys! So yes, I published `atom-content-model` today and have bumped the version in [this PR](https://github.com/guardian/atom-maker/pull/27) but for some reason it is timing out. Any idea why?

**Comment by regiskuckaertz** [comment]
(I just ran statistics and the JVM phase is taking a huge time)

File: public/js/components/AtomEdit/CustomEditors/GuideEditor.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
+import FormFieldImageSelect from '../../FormFields/FormFieldImageSelect';
+import FormFieldTextInput from '../../FormFields/FormFieldTextInput';
+import FormFieldArrayWrapper from '../../FormFields/FormFieldArrayWrapper';
+import {GuideItem} from './GuideFields/GuideItem';
+import {ManagedField, ManagedForm} from '../../ManagedEditor';
+import {atomPropType} from '../../../constants/atomPropType';
+
+export class GuideEditor extends React.Component {
```
Comment (LATaylor-guardian) [inline public/js/components/AtomEdit/CustomEditors/GuideEditor.js]: Wahey, he's using React!

**Comment by LATaylor-guardian** [comment]
👍 (once build passes)

**Comment by tomrf1** [comment]
This branch is now working in CODE, except profile atoms do not work because of a bug in atom-maker. This will be fixed later

**Pull request merged**